### PR TITLE
Ports a few Dropship PRs

### PR DESCRIPTION
**Pull request by private-tristan**
# About the pull request
ports PRs:
[Dropship sidedoors ](https://github.com/cmss13-devs/cmss13/pull/4995)
[Pried dropship's hatches cannot be locked again ]( https://github.com/cmss13-devs/cmss13/pull/5808)
[Lockdown actually closes doors on DS ](https://github.com/cmss13-devs/cmss13/pull/5015)
[Dropship doors can now always be broken by queen ](https://github.com/cmss13-devs/cmss13/pull/5917)

and also updates the midway/cyclone to have their own double doors

# Explain why it's good for the game

I like dropship sidedoors a lot.

# Testing Photographs and Procedure
messed around on a private server, made sure the dirs actually worked, loaded each shuttle from template. I did notice that the dropship lockdown buttons (the door controls) don't actually work **unless it's the normandy** and im not sure why.

# Changelog

:cl: ihatethisengine, doubleumc, private-tristan
add: Ported doubledoors from main CM
/:cl:

### CODE REVIEW

**Comment by FslashN** [comment]
The id on the dropship lockdown button probably has the wrong ID, mismatched to the doors used on the Alamo or whatever. I noticed this bug on CM a few months ago.

File: code/game/machinery/doors/multi_tile.dm [hunk 1]
```diff
@@ -275,6 +326,23 @@
...
+
+/obj/structure/machinery/door/airlock/multi_tile/almayer/dropshiprear/dropshipside/ds1/midway
+	name = "\improper Midway crew hatch"
+
+/obj/structure/machinery/door/airlock/multi_tile/almayer/dropshiprear/dropshipside/ds2
+	name = "\improper Normandy crew hatch"
+	icon = 'icons/obj/structures/doors/dropship2_side2.dmi'
+
+/obj/structure/machinery/door/airlock/multi_tile/almayer/dropshiprear/dropshipside/ds2/cyclone
+	name = "improper Cyclone crew hatch"
```
Comment (AndroBetel) [inline code/game/machinery/doors/multi_tile.dm:344]: womp womp

**Comment by AndroBetel** [comment]
i'm gonna testmerge this RED after you fix that one tiny little mistake i've spotted,

**Comment by AndroBetel** [comment]
TM'd on red

**Comment by FslashN** [comment]
Any issues with this TM'd? If not, I think we can merge it in.

**Pull request merged**